pombola/south_africa/static/sass/_person-write.scss

```diff
@@ -111,6 +111,11 @@
       background-color: white;
       padding: 2px;
     }
+
+    // Hide the Google Recaptcha element so that the 'Edit' and 'Submit' buttons align
+    #edit-write-to-message + div {
```

Comment:
might this hide the popup when it doesn't work invisibly? could we rather fix the positioning by some other means, e.g. putting something in a container?